lib/round.rb

```diff
@@ -0,0 +1,41 @@
+class Round
+  attr_reader :deck,
+              :turns
+
+  def initialize(deck)
+    @deck = deck
+    @turns = []
+  end
+
+  def current_card
+    @deck.cards[0]
+  end
+
+  def take_turn(guess)
+    @turn = Turn.new(guess, current_card)
+    @turns << @turn
+    @deck.cards.rotate!
+    @turn
+    #change the current card to the next card in the deck
+  end
+
+  def number_correct
+    @turns.count do |turn|
```

Comment:
I love this, most people don't realize you can pass a block to the `count` method. This keeps your code nice and clean.